### PR TITLE
feat: Repository migration

### PR DESCRIPTION
**Pull request by Raffy23**
## This PR
- Adds a LICENSE file, keptn-contrib GitHub automation workflows (Release, Integration tests ...)
- Adds renovate, reviewdog configuration files
- Includes a compatibility matrix in the README.md file

### CODE REVIEW

File: .versionrc.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "tagPrefix": "",
+  "preMajor": true,
```
Comment (christian-kreuzberger-dtx) [inline .versionrc.json]: we can set this to false

File: .github/workflows/integration_tests.yml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        run: |
+          echo "Failed to run integration tests!"
+          echo '{"Test": "TestGitHub Pipeline", "Action": "fail"}' >> $TEST_REPORT_FILENAME
+
+      - name: Run integration tests
+        env:
+          KEPTN_ENDPOINT: ${{ steps.install_keptn.outputs.KEPTN_API_URL }}
+          KEPTN_API_TOKEN: ${{ steps.install_keptn.outputs.KEPTN_API_TOKEN }}
+        shell: bash
+        run: gotestsum ./test/e2e/...
```
Comment (christian-kreuzberger-dtx) [inline .github/workflows/integration_tests.yml]: I don't think we have any e2e tests in the repo right now, and it doesn't look like there ever will be.
Maybe we can remove the integration tests workflow?

**Review by christian-kreuzberger-dtx — CHANGES_REQUESTED** [review]
Left a comment. 
Also I saw that [chart/charts/common-0.1.0.tgz](https://github.com/keptn-contrib/helm-service/pull/2/files#diff-0e77090ed6a390364a96b3d8789624cb06b7ca312f030a745bc335ed4aabe50b) is in the PR... not sure if that is on purpose?

File: skaffold.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ build:
   local:
     useBuildkit: true
   artifacts:
-    - image: keptndev/helm-service
+    - image: keptn-contrib/helm-service
```
Comment (christian-kreuzberger-dtx) [inline skaffold.yaml]: This might need to be
```suggestion
    - image: ghcr.io/keptn-contrib/helm-service
```

File: skaffold.yaml [hunk 4]
```diff
@@ -20,9 +20,8 @@ deploy:
         # upgradeOnChange: true
         # recreatePods: false # don't recreate all pods
         artifactOverrides:
-          image: keptndev/helm-service
-        overrides:
-          distributor:
-            image:
-              tag: 0.13.0-dev
+          helmservice:
+            image: keptn-contrib/helm-service
```
Comment (christian-kreuzberger-dtx) [inline skaffold.yaml]:
```suggestion
            image: ghcr.io/keptn-contrib/helm-service
```

File: .github/workflows/CI.yml [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+jobs:
+  prepare_ci_run:
+    name: Prepare CI Run
+    # Prepare CI Run looks at what has been changed in this commit/PR/... and determines which artifacts should be
+    # built afterwards (in other jobs that depend on this one).
```
Comment (mowies) [inline .github/workflows/CI.yml]: Is this really the case?

File: installer/manifests/common/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
```
Comment (mowies) [inline installer/manifests/common/Chart.yaml]: Isn't it weird to have the common chart in multiple places? (helm/jmeter service)

**Comment by Raffy23** [comment]
> Left a comment. Also I saw that [chart/charts/common-0.1.0.tgz](https://github.com/keptn-contrib/helm-service/pull/2/files#diff-0e77090ed6a390364a96b3d8789624cb06b7ca312f030a745bc335ed4aabe50b) is in the PR... not sure if that is on purpose?

It was like this in the original repository, I removed the common chart and used the similar helpers that we have in the job-executor-service Chart.

File: installer/manifests/common/Chart.yaml (hunk 6 above)
Comment (Raffy23) [reply installer/manifests/common/Chart.yaml]: The common chart isn't published anywhere as far as I can tell, therefore to be able to use it in the build pipeline it has to be duplicated in a few places. But for now I removed it completely

File: .github/workflows/CI.yml (hunk 5 above)
Comment (Raffy23) [reply .github/workflows/CI.yml]: The CI workflow is from the keptn-service-template-go: https://github.com/keptn-sandbox/keptn-service-template-go/blob/main/.github/workflows/CI.yml#L23-L25.

Comment (mowies) [reply .github/workflows/CI.yml]: yes, but it still doesn't check for changed files and also doesn't determine if something should be built from what I can tell

File: .gitignore [hunk 7]
```diff
@@ -17,6 +17,5 @@ MANIFEST
 vendor/*
 .DS_Store
 
-#binary - go build
-helm-service
```
Comment (christian-kreuzberger-dtx) [inline .gitignore:21]: I would also add the binary file in .gitignore - e.g., when you build it locally (for whatever reason)

Comment (Raffy23) [reply .gitignore:21]: I deleted double entries from the .gitignore file, the `helm-service` entry should be at the right location again

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**